File: c/he co so 3.cpp
#include <stdio.h>
#include <string.h>

int main(){
	int t;
	scanf("%d", &t);
	while(t--){	
		char a[20];
		getchar();
		gets(a);
		int check = 1;
		for(int i = 0; i < strlen(a); i++){
			if(a[i] != '0' && a[i] != '1' && a[i] != '2'){
				check = 0;
				break;
			}
		}
		if(check) printf("YES\n");
		else printf("NO\n");
	}
	return 0;
}

File: python/HeCoSo3.py
def check(s):
    for i in range(len(s)):
        if(s[i] != '0' and s[i] != '1' and s[i] != '2'): return False
    return True

if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        s = input()
        if check(s): print("YES")
        else: print("NO")